test/code/scxsystemlib/disk/diskpal_test.cpp

```diff
@@ -1289,7 +1289,7 @@ class SCXStatisticalDiskPalSanityTest : public CPPUNIT_NS::TestFixture
             L"tmpfs",
             L"udfs", L"usbfs",
 #if defined(linux)
-            L"udev",
+            L"udev", L"efivarfs", L"fuse.lxcfs",
 #endif
```

Comment:
See above. This tends to be a 100% copy of the list in the first file, as I recall.